botx/client/stickers_api/edit_sticker_pack.py

```diff
@@ -0,0 +1,86 @@
+from typing import List, Optional
+from uuid import UUID
+
+from typing_extensions import Literal  # For python 3.7 support
+
+from botx.client.authorized_botx_method import AuthorizedBotXMethod
+from botx.models.api_base import UnverifiedPayloadBaseModel, VerifiedPayloadBaseModel
+from botx.models.stickers import Sticker, StickerPack
+
+
+class BotXAPIEditStickerPackRequestPayload(UnverifiedPayloadBaseModel):
+    sticker_pack_id: UUID
+    name: str
+    preview: UUID
+    sticker_order_id: Optional[List[UUID]]
+
+    @classmethod
+    def from_domain(
+        cls,
+        sticker_pack_id: UUID,
+        name: str,
+        preview: UUID,
+        sticker_order_id: Optional[List[UUID]],
+    ) -> "BotXAPIEditStickerPackRequestPayload":
+        return cls(
+            sticker_pack_id=sticker_pack_id,
+            name=name,
+            preview=preview,
+            stickers_order=sticker_order_id,
+        )
+
+
+class BotXAPIGetStickerResult(VerifiedPayloadBaseModel):
+    id: UUID
+    emoji: str
+    link: str
+
+
+class BotXAPIEditStickerPackResult(VerifiedPayloadBaseModel):
+    id: UUID
+    name: str
+    public: bool
+    stickers_order: Optional[List[UUID]]
+    stickers: List[BotXAPIGetStickerResult]
+
+
+class BotXAPIEditStickerPackResponsePayload(VerifiedPayloadBaseModel):
+    status: Literal["ok"]
+    result: BotXAPIEditStickerPackResult
+
+    def to_domain(self) -> StickerPack:
+        return StickerPack(
+            id=self.result.id,
+            name=self.result.name,
+            is_public=self.result.public,
+            stickers=[
+                Sticker(id=sticker.id, emoji=sticker.emoji, image_link=sticker.link)
+                for sticker in self.result.stickers
+            ],
+        )
+
+
+class EditStickerPackMethod(AuthorizedBotXMethod):
+    status_handlers = {
```

Comment:
You should use define `status_handlers` only if you want to override it.

Reply:
Fix